src/components/shared/chart/Viewport.css

```diff
@@ -13,7 +13,6 @@
   border-top: 1px solid var(--grey-30);
   border-bottom: 1px solid var(--grey-30);
   margin-bottom: 5px;
-  cursor: -webkit-grab;
```

Comment:
I think this was removed by the rule [value-no-vendor-prefix](https://stylelint.io/user-guide/rules/list/value-no-vendor-prefix/). Note it's not added by autoprefixer because recent versions of safari support the unprefixed version (which is a good example that using autoprefixer cleans this old stuff for us).